dh_segment/utils/params_config.py

```diff
@@ -13,7 +13,7 @@ class PredictionType:
     MULTILABEL = 'MULTILABEL'
 
     @classmethod
-    def parse(cls, prediction_type):
+    def parse(cls, prediction_type) -> 'PredictionType':
```

Comment:
Unrelated to this diff, but I wonder if we use `Enum` we don't get a parse method for free?